qgis4/ansible/ansible.cfg

```diff
@@ -0,0 +1,6 @@
+[defaults]
+# Note: in case of using vagrant, the inventory will
+# be overriden by an autogenerated one.
+inventory=hosts
+roles_path=./roles/
+
```

Comment:
Is this file necessary? The `roles` directory is in the same directory as the playbook file (`playbook.yml`), so I believe setting `roles_path` is not necessary. And  I don't see the goal of setting `inventory` either.

Reply:
I had a strange bug on another playbook yesterday, where the roles where bootstrapped in the playbooks/ directory, but it would worth a try.